assets/scss/modules/_buttons.scss

```diff
@@ -78,6 +78,27 @@
       pointer-events: none;
       opacity: 0.4;
     }
+
+    &.button--has-icon {
+      width: 36px;
+      height: 36px;
+      border: 1px solid $dp-border-grey;
+      border-radius: 0;
+      font-size: 22px;
```

Comment:
I think we have font sizes in variables for this. Could you check?

Reply:
Yeah, missed that one. @fcoronelmakingsense commented the same; I'm pushing the changes right away! :)